examples/ignition/torus.yaml

```diff
@@ -0,0 +1,97 @@
+---
+ignition_version: 1
+systemd:
+  units:
+    - name: etcd3.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=etcd3
+        Conflicts=etcd2.service
+        After=docker.service
+        Requires=docker.service
+        [Service]
+        ExecStartPre=/usr/bin/mkdir -p /var/lib/etcd3
+        ExecStart=/usr/bin/docker run \
+          --net=host \
+          -e ETCD_NAME={{.etcd_name}} \
+          -e ETCD_ADVERTISE_CLIENT_URLS=http://{{.ipv4_address}}:2379 \
+          -e ETCD_INITIAL_ADVERTISE_PEER_URLS=http://{{.ipv4_address}}:2380 \
+          -e ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379 \
+          -e ETCD_LISTEN_PEER_URLS=http://{{.ipv4_address}}:2380 \
+          -e ETCD_INITIAL_CLUSTER={{.etcd_initial_cluster}} \
+          -e ETCD_DATA_DIR=/var/lib/etcd3 \
+          quay.io/coreos/etcd:v3.0.0-beta.0
+        Restart=always
+        RestartSec=10
+        TimeoutStartSec=0
+        [Install]
+        WantedBy=multi-user.target
+    - name: torus.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=CoreOS Torus
+        Requires=etcd3.service
+        After=etcd3.service
+        Requires=docker.service
+        After=docker.service
+        [Service]
+        ExecStartPre=/usr/bin/mkdir -p /srv/torus
+        ExecStart=/usr/bin/docker run \
+          --net=host \
+          -v /srv/torus:/data \
+          -e STORAGE_SIZE={{.torus_storage_size}} \
+          -e LISTEN_HOST={{.ipv4_address}} \
+          -e ETCD_HOST=127.0.0.1 \
+          -e ETCD_PORT=2379 \
+          -e AUTO_JOIN=1 \
+          -e DEBUG_INIT=1 \
```

Comment:
I think DEBUG_MKFS was from before release and the correct setting is DEBUG_INIT now